build/debian-hyperkube-base/Makefile

```diff
@@ -19,11 +19,11 @@
 
 REGISTRY?=gcr.io/google-containers
 IMAGE?=debian-hyperkube-base
-TAG=0.8
+TAG=0.8.1
```

Comment:
we don't actually use debian-hyperkube-base 0.8 anywhere, so we don't really need to push a new image for this. I could drop changes here entirely, but that was more effort.